packages/donations-app/app/components/project/components/singleGoalComponent.html

```diff
@@ -1,24 +1,24 @@
 <div class="col-md-12 col-lg-12 aliceGoal__container"
-     style="background: linear-gradient(rgba(0, 0, 0, 0.65), rgba(0, 0, 0, 0.65)), url('{{goal.image}}'); background-size: cover; background-position: center;">
+     style="background: linear-gradient(to right, rgba(0, 0, 0, 0.8), rgba(0, 0, 0, 0.0)), url('{{goal.image}}'); background-size: cover; background-position: center;">
   <div class="aliceGoal__goal"
        ng-mouseover="hoverOn(index)"
        ng-mouseleave="hoverOff()">
      <div class="aliceGoal__goal-left"
           ng-class="{ 'active' : (activeGoal !== index)}">
         <h6 class="white">Goal {{ (index + 1)}}</h6>
         <h5 class="white">{{ goal.title }}</h5>
+        <p class="white" ng-if="goal.totalPercentage > 0">We spent <span style="font-weight: bold;">{{ goal.totalValidatedForOutcome | money}}</span> to complete</p>
```

Comment:
Wouldn't it be better to have ng-if connected to the goal.totalValidatedForOutcome variable?
[to avoid not showing validated data if percentage is rounded to 0]